contrib/auto_explain/auto_explain.c

```diff
@@ -324,6 +326,8 @@ explain_ExecutorEnd(QueryDesc *queryDesc)
 		if (msec >= auto_explain_log_min_duration)
 		{
 			ExplainState es;
+			MemoryContext oldcxt =
```

Comment:
usually postgres names old context by `oldcontext`

Reply:
> usually postgres names old context by `oldcontext`

'oldcxt' is used here because the same name is used in this file in the same situation https://github.com/arenadata/gpdb/blob/7a32da0f04d66820967494ad3c177960d4931794/contrib/auto_explain/auto_explain.c#L253